legacy/application/forms/GeneralPreferences.php

```diff
@@ -150,6 +150,21 @@ public function init()
         ]);
         $this->addElement($podcast_auto_smartblock);
 
+        $scheduleTrimOverbooked = new Zend_Form_Element_Checkbox('scheduleTrimOverbooked');
+        $scheduleTrimOverbooked->setDecorators([
+            'ViewHelper',
+            'Errors',
+            'Label',
+        ]);
+        $displayScheduleTrimOverbookedValue = Application_Model_Preference::getScheduleTrimOverbooked();
+        if ($displayScheduleTrimOverbookedValue == '') {
+            $displayScheduleTrimOverbookedValue = false;
+        }
+        $scheduleTrimOverbooked->addDecorator('Label', ['class' => 'enable-tunein']);
```

Comment:
Why is there a enable-tunein class here?

Reply:
Copy/Paste from the 'Login disable' checkbox that was already there.